### PR TITLE
feat: add 1 openllm llama2 notebook demo

### PR DESCRIPTION
**Pull request by xianml**
preview in https://colab.research.google.com/drive/1NLFaFNxNxVgLPVgRevGv3NeyzJadNw96#scrollTo=QtzO813w9pR4&uniqifier=7

### CODE REVIEW

**Comment by xianml** [comment]
URGENT
@parano @aarnphm pls help review this demo. 
And this one will as template for others notebook demos. if got merged, other will follow this pattern

**Comment by xianml** [comment]
hi @aarnphm , could you help on this PR?

File: examples/openllm-llama2-demo/openllm_llama2_demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,753 @@
...
...Q6MzOTZ4cQ805ERETMmDGjdevWRLRw4UKRTQMwBNAAAAAAAAAAAABlpgI6JSVFR0eHiDQ1NYlozJixKSklqIBevHixkZFhYmKiMmVOS9u1axcRtWnThpdp3bq1g4PDy8roixeJaN26dZIk9e/fv1y5clw3fePGjZYtW/r7+0uStHHjxmvXrnHbDXd3908++QQBNOSDABoAAAAAAAAAAKBsVED37dtXkqTr16+HhYWFh4eHhoYmJydzD+ioqKjiBNBeXl5EVKNGDTs7u3LlzImoWfPmf8UrZxqUJCk4ONjS0rJGjRpdunTR09Pv1q0bPysm5o6tbdUKFSq0adPGwMDAxcXl6dOnnIkT0RdffNGxQwctLa193t6YhBDyQQANAAAAAAAAAABQqgNoToGHDRtmamo6fvz46dOnz/jbtGnTOnfubGxs/OjRo6LDX/7VpUuXxqqMHDlyxowZR48eFR0zuKVGTEzMjBkzvv76602bNqempoonxsXFzZkzp3///nPnzk1OTuanZGdnr1ixYuDAgcOGDeOaaLSBhnwQQAMAAAAAAAAAAJTqAJpT3cTExAEDBtjb2zs5OTn8zdHRsW7duj4+Pm8c/nJDZ/5ZdHbO99IF11zoayF9hoIQQAMAAAAAAAAAAJTqAPotys3NVfwtRyXfAnl5efx4vl+Jx7kcW+D1KBSKIoqv4UOGABoAAAAAAAAAAKBsBNCKV0D4C6UWAmgAAAAAAAAAAICyEUADlDkIoAEAAAAAAAAAAIoDATRAiSGABgAAAAAAAAAAKA4E0AAlhgAaAAAAAAAAAACgOBBAA5QYAmgAAAAAAAAAAIDiQAANUGIIoAEAAAAAAAAAAIoDATRAiSGABgAAAAAAAAAAKA4E0AAlhgAaAAAAAAAAAACgOBBAA5QYAmgAAAAAAAAAAIDiQAANUGIIoAEAAAAAAAAAAIoDATRAiSGABgAAAAAAAAAAKA4E0AAlhgAaAAAAAAAAAACgOBBAA5TVADo3N/cBAAAAAAAAAADAv+zevXtZWVlvFmEhgAYoqwF0Xl7eCwAAAAAAAAAAgH9ZcnJyTk7Om0VYCKABymoADQAAAAAAAAAAUMohgAYoMQTQAAAAAAAAAAAAxYEAGqDEEEADAAAAAAAAAAAUBwJogBJDAA0AAAAAAAAAAFAcCKABSgwBNAAAAAAAAAAAQHEggAYoMQ
```
Comment (aarnphm) [inline examples/openllm-llama2-demo/openllm_llama2_demo.ipynb:634]:
```suggestion
        "!openllm list-bentos"
```

**Review by aarnphm — CHANGES_REQUESTED** [review]
ugh ignore the code suggest. But you can use `openllm list-bentos` for the built bentos instead of `bentoml list -o json`

**Comment by xianml** [comment]
> ugh ignore the code suggest. But you can use `openllm list-bentos` for the built bentos instead of `bentoml list -o json`

updated

**Review by aarnphm — APPROVED** [review]

**Pull request merged**